subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -54,7 +54,7 @@
         .changedTo(68, "5.0-milestone-1")
         .changedTo(69, "5.0-rc-1")
         .changedTo(71, "5.3-rc-1")
-        .changedTo(76, "6.0-rc-1")
+        .changedTo(77_123_456, "6.0-rc-1")
```

Comment:
The weird version change is intentional here, this will be set to `77` (or whatever n+1 version) just before merging to `master`.